### PR TITLE
Adds simulated sensor

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Docs for simulated sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/12539) (if applicable):** home-assistant/home-assistant#<12539>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.simulated.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+To give an example of simulating real world data, a simulated relative humidity sensor (in %) can be added using the following config:
+
+```yaml
+- platform: simulated
+  name: 'simulated relative humidity'
+  unit: '%'
+  amplitude: 0 # Turns off the periodic contribution
+  mean: 50
+  spread: 10
```
Comment (c727) [inline source/_components/sensor.simulated.markdown]: Spread is missing below

File: source/_components/sensor.simulated.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+```yaml
+sensor:
+  - platform: simulated
+```
+
+To give an example of simulating real world data, a simulated relative humidity sensor (in %) can be added using the following config:
+
+```yaml
+- platform: simulated
```
Comment (c727) [inline source/_components/sensor.simulated.markdown]: Please also include `sensor:` here

File: source/_components/sensor.simulated.markdown (hunk 1 above)
Comment (robmarkcole) [reply source/_components/sensor.simulated.markdown]: ok

File: source/_components/sensor.simulated.markdown (hunk 2 above)
Comment (robmarkcole) [reply source/_components/sensor.simulated.markdown]: ok

**Pull request merged**